File: RokidSDKDemoJava/app/src/main/java/com/rokid/mobile/sdk/demo/java/webkit/DemoWebChromeClient.java
package com.rokid.mobile.sdk.demo.java.webkit;

import android.webkit.WebView;

import com.rokid.mobile.sdk.webkit.SDKWebChromeClient;
import com.rokid.mobile.webview.lib.RKWebBridge;

/**
 * Description: TODO
 * Author: Shper
 * Version: V0.1 2017/12/6
 */
public class DemoWebChromeClient extends SDKWebChromeClient {

    public DemoWebChromeClient(RKWebBridge webBridge) {
        super(webBridge);
    }

    @Override
    public void onProgressChanged(WebView view, int newProgress) {
        super.onProgressChanged(view, newProgress);
    }

}

File: RokidSDKDemoKotlin/app/src/main/kotlin/com/rokid/mobile/sdk/demo/webkit/DemoWebChromeClient.kt
package com.rokid.mobile.sdk.demo.webkit

import android.webkit.WebView
import com.rokid.mobile.sdk.webkit.SDKWebChromeClient
import com.rokid.mobile.webview.lib.RKWebBridge

/**
 * Description: TODO
 * Author: Shper
 * Version: V0.1 2018/2/26
 */
class DemoWebChromeClient(bridge: RKWebBridge) : SDKWebChromeClient(bridge) {

    override fun onProgressChanged(view: WebView?, newProgress: Int) {
        super.onProgressChanged(view, newProgress)
    }

}